lib/matplotlib/mlab.py

```diff
@@ -2867,14 +2867,19 @@ def get_func(name, item, func):
         'print':  'print_',
         }
 
-    def get_converters(reader):
+    def get_converters(reader, comments):
```

Comment:
Ah, I just now notice that. And I also now see that this is not a public function, so `comments` doesn't need to be optional.